spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -860,7 +871,35 @@ public interface TraderRepository extends CrudRepository<Trader, String> {
 
 ==== Projections
 
-This feature is not supported yet.
+Spring Data Cloud Datastore supports {spring-data-commons-ref}/#projections[projections]. You can define projection interfaces based on domain types and add query methods that return them in your repository:
+
+[source, java]
+----
+public interface TradeProjection {
+
+	String getAction();
+
+	@Value("#{target.symbol + ' ' + target.action}")
+	String getSymbolAndAction();
+}
+
+public interface TradeRepository extends DatastoreRepository<Trade, Key> {
+
+	List<Trade> findByTraderId(String traderId);
+
+	List<TradeProjection> findByAction(String action);
+
+	@Query("SELECT action, symbol FROM trades WHERE action = @action")
+	List<TradeProjection> findByQuery(String action);
+}
+----
+
+Projections can be provided by name-convention-based query methods as well as by custom SQL queries.
+If using custom SQL queries, you can further restrict the fields retrieved from Datastore to just those required by the projection.
```

Comment:
Mention that the whole Datastore Entity is retrieved in case of queries based on method name.